Common/ILEdits.cs

```diff
@@ -15,18 +15,19 @@ public static class ILEdits
     {
         public static void Load()
         {
-            IL.Terraria.Main.UpdateAudio += TitleMusic;
-            IL.Terraria.Main.UpdateAudio += TitleDisable;
-            IL.Terraria.Projectile.VanillaAI += GrappleCollision;
+            // TODO: There is literally zero percent chance this works the same in 1.4. Needs to be redone.
+            // IL.Terraria.Main.UpdateAudio += TitleMusic;
+            // IL.Terraria.Main.UpdateAudio += TitleDisable;
+            // IL.Terraria.Projectile.VanillaAI += GrappleCollision;
             //IL.Terraria.Liquid.Update += UpdateWater;
             //IL.Terraria.Liquid.QuickWater += QuickWater;
         }
 
         public static void Unload()
         {
-            IL.Terraria.Main.UpdateAudio -= TitleMusic;
-            IL.Terraria.Main.UpdateAudio -= TitleDisable;
-            IL.Terraria.Projectile.VanillaAI -= GrappleCollision;
+            // IL.Terraria.Main.UpdateAudio -= TitleMusic;
+            // IL.Terraria.Main.UpdateAudio -= TitleDisable;
+            // IL.Terraria.Projectile.VanillaAI -= GrappleCollision;
```

Comment:
I haven't _really_ tested these. I'm expecting that they will just fail. Best to read the code and manually verify that the assumptions continue to make sense. I really don't know enough about the underlying assumptions here to do that easily.